packages/fast-components/src/dialog/dialog.template.ts

```diff
@@ -0,0 +1,33 @@
+import { html } from "@microsoft/fast-element";
+import { when } from "@microsoft/fast-element";
+import { ref } from "@microsoft/fast-element/dist/directives/";
+import { Dialog } from "./dialog";
```

Comment:
nit: consolidate imports from `@microsoft/fast-element`

Reply:
These aren’t all exported from the root - @EisenbergEffect on his thinking there